portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,30 +1,25 @@
 package com.google.sps.data;
 
 import java.util.Date;//will use later
-public class Comment{
+public class Comment {
   private String comment;
   private String posterName;
 
-  public Comment(String comment, String posterName)
-  {
+  public Comment(String comment, String posterName){
       this.comment = comment;
       this.posterName = posterName;
   }
 
-  public Comment(String comment)
-  {
+  public Comment(String comment){
       this.comment = comment;
       this.posterName = "Anonymous";
   }
 
-
-  public String getComment()
-  {
+  public String getComment(){
       return comment;
   }
 
-  public String getPosterName()
-  {
+  public String getPosterName(){
```

Comment:
In general we do 2 spaces: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation